config/database.yml

```diff
@@ -2,6 +2,8 @@ default: &default
   adapter: postgresql
   encoding: unicode
   pool: 5
+  username: nicher
```

Comment:
I don't understand the reason, why you delete default section.

If you want to have different parameters for different environments, please delete database.yml from the repo and add instead database.yml.example